Gemfile

```diff
@@ -4,13 +4,6 @@ gemspec
 gem 'rake'
 gem 'metriks', :require => false
 gem 'statsd-ruby', :require => false
-gem 'rspec'
+gem 'rspec', '~> 2.14'
 gem 'rack-test'
 gem 'activesupport', :require => false
-
-group(:guard) do
```

Comment:
I haven't been. We can add it back in later if you are doing flipper dev and want it.